mypy/test/testconfigparser.py

```diff
@@ -0,0 +1,168 @@
+import os
+import tempfile
+from unittest import TestCase, main
+
+from mypy.config_parser import parse_config_file
+from mypy.options import Options
+
+
+class TestConfigParser(TestCase):
+    def test_parse_config_file_with_single_file(self) -> None:
+        """A single file should be correctly parsed."""
+        with tempfile.TemporaryDirectory() as tmpdirname:
+            config_path = os.path.join(tmpdirname, "test_config.ini")
+
+            with open(config_path, "w") as f:
+                f.write(
+                    """
+                    [mypy]
+                    files = file1.py
+                    """
+                )
+
+            options = Options()
+
+            parse_config_file(options, lambda: None, config_path, stdout=None, stderr=None)
+
+            self.assertEqual(options.files, ["file1.py"])
```

Comment:
unittest-style checks aren't really common in the world of pytest. Let's avoid javisms.
```suggestion
            assert options.files == ["file1.py"]
```

Reply:
(unless it doesn't work here for some reason)